.travis.yml

```diff
@@ -8,12 +8,15 @@ addons:
     sources:
     - google-chrome
     packages:
-    - google-chrome-stable
+    - google-chrome-stable xvfb
```

Comment:
Or is it to install "xvfb" ? I think it's pre-installed on travis VMs.
(as a side note, I think the travis syntax is one package per line)